src/react/hooks/useLazyQuery.ts

```diff
@@ -25,6 +25,41 @@ const EAGER_METHODS = [
   "subscribeToMore",
 ] as const;
 
+/**
+ * A hook for imperatively executing queries in an Apollo application, e.g. in response to user interaction.
+ *
+ * > Refer to the [Queries - Manual execution with useLazyQuery](https://www.apollographql.com/docs/react/data/queries#manual-execution-with-uselazyquery) section for a more in-depth overview of `useLazyQuery`.
+ *
+ * @example
+ * ```jsx
+ * import { gql, useLazyQuery } from "@apollo/client";
+ *
+ * const GET_GREETING = gql`
+ *   query GetGreeting($language: String!) {
+ *     greeting(language: $language) {
+ *       message
+ *     }
+ *   }
+ * `;
+ *
+ * function Hello() {
+ *   const [loadGreeting, { called, loading, data }] = useLazyQuery(
+ *     GET_GREETING,
+ *     { variables: { language: "english" } }
+ *   );
+ *   if (called && loading) return <p>Loading ...</p>
+ *   if (!called) {
+ *     return <button onClick={() => loadGreeting()}>Load greeting</button>
+ *   }
+ *   return <h1>Hello {data.greeting.message}!</h1>;
+ * }
+ * ```
+ * @since 3.0.0
+ *
+ * @param query - A GraphQL query document parsed into an AST by `gql`.
+ * @param options - Default options to control how the query is executed.
+ * @returns A tuple in the form of `[executeLazyQuery, queryResult]`
```

Comment:
```suggestion
 * @returns A tuple in the form of `[execute, result]`
```

It looks like you've named these keys `execute` and `result` ([reference](https://github.com/apollographql/apollo-client/pull/11381/files?diff=unified&w=1#diff-61523aec6529faa71ab822ab03e00aa48743ebf5c156794ed907f2765e00d5f2R322-R323)). Can we update to match?